OpenDreamRuntime/Procs/ProcDecoder.cs

```diff
@@ -126,6 +127,9 @@ public struct ProcDecoder {
             case DreamProcOpcode.PushType:
             case DreamProcOpcode.DebugLine:
             case DreamProcOpcode.MassConcatenation:
+            case DreamProcOpcode.JumpIfNullNoPop:
+            case DreamProcOpcode.JumpIfTrueReference:
+            case DreamProcOpcode.JumpIfFalseReference:
```

Comment:
These two should go with `JumpIfNullDereference` below.

`Initial` also no longer takes an operand and should be removed from this switch.